src/pages/Admin/AdminVenueRoomDetails.tsx

```diff
@@ -115,18 +117,15 @@ export const AdminVenueRoomDetails = ({
                   }
                 </div>
                 <div className="toggle-room">
-                  <label id={"toggle-" + index} className="switch">
-                    <input
-                      type="checkbox"
-                      id={"toggle-" + index}
-                      name={"toggle-" + index}
-                      checked={room.isEnabled}
-                      onClick={() => {
-                        updateRoom(!room.isEnabled);
-                      }}
-                    />
-                    <span className="slider round"></span>
-                  </label>
+                  <Toggler
+                    labelClassName="switch"
+                    id={"toggle-" + index}
+                    name={"toggle-" + index}
+                    defaultToggled={room.isEnabled}
+                    onToggle={() => {
+                      updateRoom(!room.isEnabled);
+                    }}
+                  />
                   <div>Turn room {room.isEnabled ? "Off" : "On"}</div>
```

Comment:
Label should be passed inside the component. Remove labelClassName

Reply:
done 3c68fdfc66a9346cd67e1d15bd5fe7a688d52090 + 7983b12912a95f067c32c1fc6ec8d47716e372d0